util/reward-calculator/src/lib.rs

```diff
@@ -33,7 +34,10 @@ impl<'a, P: ChainProvider> RewardCalculator<'a, P> {
 
     /// `RewardCalculator` is used to calculate block finalize target's reward according to the parent header.
     /// block reward consists of four parts: base block reward, tx fee, proposal reward, and secondary block reward.
-    pub fn block_reward(&self, parent: &Header) -> Result<(Script, Capacity), FailureError> {
+    pub fn block_reward(
+        &self,
+        parent: &Header,
+    ) -> Result<(Script, Capacity, BlockReward), FailureError> {
```

Comment:
We can impl a method on `BlockReward` to return the total capacity, then we don't need to return the total capacity here.

Reply:
or add a new field `total`?